### PR TITLE
Slick slim

### PR DESCRIPTION
**Pull request by macdonaldr93**
1.6.1 - Slick Slim

| Version | File size | Gzip |
| --- | --- | --- |
| Slick | 42kb | 10kb |
| Slick slim | 32kb | 8kb |
### Options
- adaptiveHeight
- responsive
- variableWidth
- vertical
- verticalSwiping
- centerMode
- infinite
### Methods
- addSlide
- animateHeight
- removeSlide
- checkResponsive
- registerBreakpoints
- setHeight
- updateArrows

### CODE REVIEW

**Comment by macdonaldr93** [comment]
@chrisberthe @cshold @stevebosworth @t-kelly @richgilbank ^^

File: slick/slick.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
 |___/_|_|\___|_|\_(_)/ |___/
                    |__/
 
- Version: 1.6.0
+ Version: 1.6.1
```
Comment (chrisberthe) [inline slick/slick.js]: I don't know why I chuckled at this.

**Comment by chrisberthe** [comment]
Nice 👍

**Comment by macdonaldr93** [comment]
I think if we go this path, we have to completely rename the slider so that merchants / developers don't associate it to Slick. Or, at least make that obvious in the README.

**Comment by cshold** [comment]
Thoughts on creating a slick-slim.js file? That way if we need any of the other things in the future we don't have to re-add it in? Better yet, do these changes just in Debut?

**Comment by macdonaldr93** [comment]
Makes sense to me. Maybe I'll just throw that changelog in Debut?

**Pull request closed without merging**